dotCMS/src/main/java/com/dotmarketing/util/TaskLocatorUtil.java

```diff
@@ -5,6 +5,7 @@
 import java.util.concurrent.CopyOnWriteArrayList;
 
 import com.dotcms.repackage.com.google.common.collect.ImmutableList;
+import com.dotcms.repackage.javax.ws.rs.HEAD;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.repackage.javax.ws.rs.HEAD'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)